src/main/java/ddingdong/ddingdongBE/domain/form/service/FacadeCentralFormServiceImpl.java

```diff
@@ -235,6 +247,10 @@ private void validateDuplicationDateExcludingSelf(
         }
     }
 
+    private void validateEndDate(LocalDate startDate, LocalDate endDate) {
+        if (startDate.isAfter(endDate)) { throw new InvalidFormEndDateException(); }
```

Comment:
```suggestion
    private void validateEndDate(LocalDate startDate, LocalDate endDate) {
        if (endDate.isBefore(startDate) || endDate.isEqual(startDate)) { 
              throw new InvalidFormEndDateException(); 
        }
```

p4) 
1. 시작일과 동일할 경우에도 예외를 발생해야하지 않을까요?
2. endDate를 검증하는 거니, endDate를 기준으로 하면 더 가독성에 좋을 것 같아요!

Reply:
가독성 부분은 생각도 못했는데...ㅎㅎ 감사합니다 수정했습니다!